### PR TITLE
[LG-5365] Add Production Configuration indicator to SP Dashboard

### PR DESCRIPTION
**Pull request by TiannaW24**
### Work Ticket
https://cm-jira.usa.gov/browse/LG-5365

### Changes

-   Add a boolean value for indicating whether a configuration is meant for production
- Persist boolean value to ServiceProviders table

### PR Checklist:

1. Have you tagged the appropriate dev(s) for review?
2. Have you moved any Jira/Smartsheet tickets to Code Review?

### CODE REVIEW

**Comment by TiannaW24** [comment]
Screen Shot 2022-01-10 at 2 33 38 PM

**Comment by TiannaW24** [comment]
Screen Shot 2022-01-10 at 2 35 31 PM

**Review by jauschalley — APPROVED** [review]
You may want to add a test for the new field in `spec/features/service_providers_spec.rb` but I won't block you over it. 🤷‍♂️

File: app/views/service_providers/_form.html.erb [hunk 1]
```diff
@@ -1,5 +1,22 @@
 <%= form.error_notification %>
 <fieldset class='usa-fieldset'>
+
+  <legend class='usa-sr-only'>Production Configuration</legend>
+  <%= form.label :admin,
```
Comment (orenyk) [inline app/views/service_providers/_form.html.erb:5]: **question:** should this be `form.label :prod_config`? My Rails form helpers / SimpleForm is a little fuzzy, so I could be off, but it seems like it should be the attribute name?

```suggestion
  <%= form.label :prod_config,
```

File: db/migrate/20220110190210_add_prod_configto_sp_table.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddProdConfigtoSpTable < ActiveRecord::Migration[6.1]
+  def change
+    add_column :service_providers, :prod_config, :boolean
```
Comment (orenyk) [inline db/migrate/20220110190210_add_prod_configto_sp_table.rb:3]: **suggestion:** do we want to set a default (presumably `false`) and also prevent null values?

File: app/views/service_providers/show.html.erb [hunk 3]
```diff
@@ -138,7 +145,7 @@
 
   <h2 class="margin-bottom-0">
     <label for="signed_response_message_requested">Signed Response Message Requested:</label> 
-    <span class="font-mono-xs margin-left-5"><%= service_provider.signed_response_message_requested? %></span>
+    <span class="font-mono-xs margin-left-5"><%= service_provider.signed_response_message_requested? ? 'Yes' : 'No' %></span>
```
Comment (orenyk) [inline app/views/service_providers/show.html.erb:148]: **praise:** big fan of this!

**Review by orenyk — APPROVED** [review]
Two small comments, otherwise LGTM!

File: db/migrate/20220110190210_add_prod_configto_sp_table.rb (hunk 2 above)
Comment (jauschalley) [reply db/migrate/20220110190210_add_prod_configto_sp_table.rb:3]: +1 on this comment

Comment (TiannaW24) [reply db/migrate/20220110190210_add_prod_configto_sp_table.rb:3]: Makes sense to me.

**Comment by TiannaW24** [comment]
> You may want to add a test for the new field in `spec/features/service_providers_spec.rb` but I won't block you over it. 🤷‍♂️

I'll leave the unit testing for when we have logic around the button

**Pull request merged**